metrics/accumulator.go

```diff
@@ -122,15 +122,26 @@ func (a *MetricDataAccumulator) volumeStats(podResource *Resource, s stats.Volum
 }
 
 func (a *MetricDataAccumulator) accumulate(r *Resource, m ...Metrics) {
-	retMetrics := make(Metrics)
+	podName := r.Name
+
+	// check if pod name already exists
+	var resourceMetrics *ResourceMetrics
+	if rm, ok := a.Data[podName]; ok {
+		// use existing entry from map
+		resourceMetrics = rm
```

Comment:
Right now this is just replacing the map entry, but I imagine we implement logic that will aggregate/merge the individual metrics appropriately instead.

Things like `network.bytes` should be a SUM. But `uptime` should be the MAX value instead.